docs/source/user-guide/latest/configs.md

```diff
@@ -81,7 +81,7 @@ These settings can be used to determine which parts of the plan are accelerated
 | `spark.comet.explain.native.enabled` | When this setting is enabled, Comet will provide a tree representation of the native query plan before execution and again after execution, with metrics. | false |
 | `spark.comet.explain.rules` | When this setting is enabled, Comet will log all plan transformations performed in physical optimizer rules. Default: false | false |
 | `spark.comet.explainFallback.enabled` | When this setting is enabled, Comet will provide logging explaining the reason(s) why a query stage cannot be executed natively. Set this to false to reduce the amount of logging. | false |
-| `spark.comet.logFallbackReasons.enabled` | When this setting is enabled, Comet will log warnings for all fallback reasons. Can be overridden by environment variable `ENABLE_COMET_LOG_FALLBACK_REASONS`. | false |
+| `spark.comet.logFallbackReasons.enabled` | When this setting is enabled, Comet will log warnings for all fallback reasons. Can be overridden by environment variable `ENABLE_COMET_LOG_FALLBACK_REASONS`. | true |
```

Comment:
https://github.com/andygrove/datafusion-comet/blob/8e97fd179be5733dad328ba7dac3308bd0a319f7/common/src/main/scala/org/apache/comet/CometConf.scala#L502 says that the default is `false`

Reply:
We really need CI checks to ensure the generated docs stay in sync with the code. I will try to get that working today.